pyocd/subcommands/rtt_cmd.py

```diff
@@ -119,13 +119,17 @@ def invoke(self) -> int:
 
                 memory_map: MemoryMap = target.get_memory_map()
                 ram_region: MemoryRegion = memory_map.get_default_region_of_type(MemoryType.RAM)
-
-                if self._args.address is None or self._args.size is None:
-                    rtt_range_start = ram_region.start
-                    rtt_range_size = ram_region.length
-                elif ram_region.start <= self._args.address and self._args.size <= ram_region.length:
-                    rtt_range_start = self._args.address
-                    rtt_range_size = self._args.size
+                def valid_range(start, size):
+                    return start != None and size != None \
+                        and ram_region.start <= start and size + start <= ram_region.start + ram_region.length
+                rtt_range_start, rtt_range_size = None, None
+                config_start, config_size = session.options.get('rtt.start'), session.options.get('rtt.size')
```

Comment:
Could you please add the new options to `pyocd/core/options.py` so they have help?